packaging/installer-linux/installer-debian/src/main/resources/common/neo4j

```diff
@@ -46,6 +47,14 @@ SCRIPTNAME=/etc/init.d/$NAME-service
 
 do_start()
 {
+  # Make sure that /var/run/neo4j exists
+  [ -d "${PIDDIR}" ] || mkdir -p "${PIDDIR}"
+
+  # And that it has correct permissions
+  if [ "${NEO_USER}" != "root" ]; then
```

Comment:
Given that we don't expect people ever to run as `root` and if they do the operation you are guarding against is a no-op, I would do without the conditional altogether.